Sources/Views/ALKChatCell.swift

```diff
@@ -113,6 +114,40 @@ public final class ALKChatCell: SwipeTableViewCell, Localizable {
     public enum Config {
         public static var iconMuted = UIImage(named: "muted", in: Bundle.km, compatibleWith: nil)
     }
+    
+    public enum PlatformIcon {
+        public static var iconFacebook = UIImage(named: "platformFacebook", in: Bundle.km, compatibleWith: nil) ?? UIImage()
+        public static var iconWhatsapp = UIImage(named: "platformWhatsapp", in: Bundle.km, compatibleWith: nil) ?? UIImage()
+        public static var iconPhone = UIImage(named: "platformPhone", in: Bundle.km, compatibleWith: nil) ?? UIImage()
+        public static var iconWeb = UIImage(named: "platformWeb", in: Bundle.km, compatibleWith: nil) ?? UIImage()
+        public static var icon360Dialog = UIImage(named: "platform360Dialog", in: Bundle.km, compatibleWith: nil) ?? UIImage()
+        public static var iconMeta = UIImage(named: "platformMeta", in: Bundle.km, compatibleWith: nil) ?? UIImage()
+        public static var iconLine = UIImage(named: "platformLine", in: Bundle.km, compatibleWith: nil) ?? UIImage()
+        public static var iconInstagram = UIImage(named: "platformInstagram", in: Bundle.km, compatibleWith: nil) ?? UIImage()
+        public static var iconMessengerFB = UIImage(named: "platformMessengerFB", in: Bundle.km, compatibleWith: nil) ?? UIImage()
+        public static var iconSunshine = UIImage(named: "platformSunshine", in: Bundle.km, compatibleWith: nil) ?? UIImage()
+        public static var iconTelegram = UIImage(named: "platformTelegram", in: Bundle.km, compatibleWith: nil) ?? UIImage()
+        public static var iconViber = UIImage(named: "platformViber", in: Bundle.km, compatibleWith: nil) ?? UIImage()
+        public static var iconTwillio = UIImage(named: "platformTwillio", in: Bundle.km, compatibleWith: nil) ?? UIImage()
+    }
+    
+    let platformIcons: [String: UIImage] = [
+        "MOBILE": PlatformIcon.iconPhone,
+        "WEB": PlatformIcon.iconWeb,
+        "FACEBOOK": PlatformIcon.iconFacebook,
+        "WHATSAPPCLOUDAPI": PlatformIcon.iconWhatsapp,
+        "WHATSAPPTWILIO": PlatformIcon.iconWhatsapp,
+        "WHATSAPPDIALOG360": PlatformIcon.iconWhatsapp,
+        "DIALOG360": PlatformIcon.icon360Dialog,
+        "ZENDESKSUNSHINE": PlatformIcon.iconSunshine,
+        "VIBER": PlatformIcon.iconViber,
+        "LINE": PlatformIcon.iconLine,
+        "META": PlatformIcon.iconMeta,
+        "TELEGRAM": PlatformIcon.iconTelegram,
+        "MESSENGERFB": PlatformIcon.iconMessengerFB,
+        "INSTAGRAM": PlatformIcon.iconInstagram,
+        "TWILIO": PlatformIcon.iconTwillio
+    ]
```

Comment:
What's the use of two enums here

Reply:
First is used for loading the image and second is for checking the keywords and setting up the image. Do I need to merge these two enums.

Reply:
yes you can merge it into single enum

Reply:
okay

Reply:
done merged it into single enum